adding more operator FAQs; corrections to node monitoring page (#169)
onflow/flow: docs/content/node-operation/faq.mdx

### Can I bootstrap and run a node at anytime?

The node has to be included as part of the spork and only then will be part of the Flow protocol. Hence, a node can be bootstrapped only during a spork.

### Would it hurt the network to have a node that constantly spins up and down?

A node needs to be part of the genesis block. The node can bounce after that and while that may not break the Flow protocol, it will cause severe degradation of overall performance.

> A way to prevent this is to check your equipment meets Flow's recommended requirements, periodically checking for updates and announcements in Discord but also using a node monitoring system for when your node does go offline.

Reviewer: "A way to prevent this is to check your equipment meets Flow's recommended requirements..."
Could you add a link to the information about requirements?

> ### Will Flow have a regular schedule for Sporks? Is this how new node operators can be on-boarded to the network?

Reviewer: ```suggestion
### Will Flow have a regular schedule for Sporks?
```
Deleting because this question is not answered in the paragraph below it.

> Yes, but the precise schedule will be somewhat variable to also include protocol upgrades. Currently, Flow has a Mainnet Spork and a Testnet Spork roughly every two weeks.

Reviewer: ```suggestion
Yes, but the precise schedule will be somewhat variable and will also include protocol upgrades. (Currently, Flow has a Mainnet Spork and a Testnet Spork roughly every two weeks).
```

### How do I update the Node Software?

One of the reason to [Spork](/node-operation/spork) is to make sure all nodes update to the latest software version. Hence, as long as you are participating in the Spork, you should have the latest software update.
However, if we do release any software update in between a Spork (e.g. an emergency patch) we will announce it on Discord.

### Can anybody run a node? What is the approval process? What is the approval process?

> For the first 6 months of the network, all participating nodes must present a case for how they will add unique value to the network. Individuals go through an application process that includes a bit about their background and experience contributing to decentralized projects.

Reviewer: ```suggestion
For the first 6 months after the main network is launched, all participating nodes must present a case for how they will add unique value to the network. There will be an application process. Applicants will be asked to submit some information to the Flow team including a bit about their background and experience contributing to decentralized projects.
```

Reviewer: This still reads a little akwardly, might need some more work
